platform/uwp/os_uwp.cpp

```diff
@@ -739,6 +739,11 @@ String OSUWP::get_environment(const String &p_var) const {
 	return "";
 };
 
+bool OSUWP::set_environment(const String &p_var, const String &p_value) const {
+
+	return false;
+}
+
```

Comment:
I left this like this because the other environment methods are not implemented either. Is there a reason for that? Shouldn't the same implementation as `OS_Windows` work for `OSUWP` in this case?

Reply:
AFAIK UWP does not have access to the environment variables. Unless that changed since I last looked.